app/validators/cycle_verification_validator.rb

```diff
@@ -5,7 +5,7 @@ def validate_each(record, attribute, _application_choice)
     record.errors.add(
       attribute,
       :can_not_submit_this_time_in_the_cycle,
-      message: "You cannot submit this application now. You will be able to submit it from #{CycleTimetable.apply_opens.to_s(:govuk_date_and_time)}.",
+      message: "You cannot submit this application now. You will be able to submit it from #{CycleTimetable.apply_opens.to_fs(:govuk_date_and_time)}.",
```

Comment:
Awesome PR! But I fixed this one on #8458

Reply:
I don't know enough about your PR to approve it. But if it gets approved first, I'll update mine accordingly.

Reply:
Merged. You can rebase now for taking this change :)